chalicelib/check_setup.json

```diff
@@ -1,438 +1,516 @@
-            "thirty_min_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "change_in_item_counts": {
         "title": "Change in item counts",
-        "group": "Metadata checks",
+        "group": "Metadata Checks",
         "schedule": {
-            "morning_checks_2": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "page_children_routes": {
         "title": "Pages with bad routes",
-        "group": "Audit checks",
+        "group": "Audit Checks",
         "schedule": {
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "workflow_properties": {
         "title": "Workflows with missing or duplicate steps properties",
-        "group": "Audit checks",
+        "group": "Audit Checks",
         "schedule": {
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "check_validation_errors": {
         "title": "Search for Validation Errors",
-        "group": "Audit checks",
-        "schedule": {
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
-                }
-            }
-        }
-    },
-    "secondary_queue_deduplication": {
-        "title": "Secondary queue deduplication",
-        "group": "System checks",
+        "group": "Audit Checks",
         "schedule": {
-            "fifteen_min_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
```

Comment:
this is another one that may still benefit from running automatically, even if the schedule is less frequent. It helps to know for instance if a recent deploy has caused unexpected validation errors, and just telling people to run this check after every time they deploy an env is easily forgotten. Fifteen minutes is overkill but maybe daily if we are doing frequent deploys.